client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -173,6 +174,24 @@ public void testExists() {
         getAndExistsTest(Request::exists, "HEAD");
     }
 
+    public void testIndicesExist_OneIndex() {
+        final String indexName = randomAlphaOfLength(10);
+        IndicesExistsRequest indicesExistRequest = new IndicesExistsRequest(indexName);
+        final Request request = Request.indicesExist(indicesExistRequest);
+        assertEquals("/" + indexName, request.getEndpoint());
+        assertEquals("HEAD", request.getMethod());
+    }
+
+    public void testIndicesExist_OneIndexExists_OneDoesnt() {
+        final String index1 = randomAlphaOfLength(10);
+        final String index2 = randomAlphaOfLength(10);
+
+        IndicesExistsRequest indicesExistRequest = new IndicesExistsRequest(index1, index2);
+        final Request request = Request.indicesExist(indicesExistRequest);
+        assertEquals("/" + index1 + "," + index2, request.getEndpoint());
+        assertEquals("HEAD", request.getMethod());
+    }
```

Comment:
I think that one test here is enough. test it with a random number of indices ?

Reply:
Done! Test is now using up to 10 indices with random names.